### PR TITLE
Query Overview v2.1.1

### LINKED ISSUE #147: Add a destroy method.
That not only clears the contents of the containing element and the Webcharts object(s), but the stylesheet as well.

# Test notes
* Open the developer console.
*  On the Elements tab expand the `<head>` element and verify that the last child node is a `<style>` element and that its styles include _.query-overview_ classes.
* On the Console tab run `d3.select('.query-overview').node()` and verify the Query Overview `<div>` element is returned.
* Run `d3.select('.wc-chart').datum().destroy()` and verify both the `<style>` element and `<div>` element are removed from the DOM.

Reference the images in [this crf-heat-map issue](https://github.com/RhoInc/crf-heat-map/issues/135) for screenshots.

### PR DESCRIPTION
**Pull request by samussiah**
# Issues
Closes #147

### CODE REVIEW

**Comment by samussiah** [comment]
@pburnsdata , in retrospect it was a terrible idea to add unit tests.  Took forever, required importing every d3 function explicitly, and was extremely frustrating.  You can ignore the explicit import file changes for the most part.

File: src/chart/onInit.js [hunk 1]
```diff
@@ -25,5 +25,5 @@ export default function onInit() {
     removeInvalidControls.call(this);
 
     //Initialize listing.
-    this.listing.init(this.raw_data);
+    this.listing.init(this.raw_data, this.listing.test);
```
Comment (pburnsdata) [inline src/chart/onInit.js:28]: @samussiah whats going on with this guy?

**Review by pburnsdata — APPROVED** [review]
Looks good! Sorry that the tests got wild - definitely a lot of replaced functions here haha

File: src/chart/onInit.js (hunk 1 above)
Comment (samussiah) [reply src/chart/onInit.js:28]: webcharts has logic that expects the containing element to have been rendered before it proceeds.  but the headless browser we use for testing doesn't actually render the page so there's some logic in `init` that avoids the rendered logic when running unit tests.

Comment (pburnsdata) [reply src/chart/onInit.js:28]: gotcha gotcha - makes sense

**Review by danedexF5 — APPROVED** [review]
Approved. DESTROY method works.

**Comment by danedexF5** [comment]
There actually seems to be an IE issue with destroy. Spencer is looking at it.

**Comment by danedexF5** [comment]
Destroy issue in IE is fixed, all good.

**Review by danedexF5 — APPROVED** [review]
approved

**Pull request merged**